src/components/Section/Section.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Section.scss';
+
+function getBackgroundColorClass(bgc) {
+  if (bgc === 'transparent') {
+    return styles.isTransparent;
+  } else if (bgc === 'white') {
+    return styles.isWhite;
+  } else if (bgc === 'grey') {
+    return styles.isGrey;
+  } else if (bgc === 'primary') {
+    return styles.isPrimary;
+  } else if (bgc === 'secondary') {
+    return styles.isSecondary;
+  } else if (bgc === 'tertiary') {
+    return styles.isTertiary;
+  } else {
+    return '';
+  }
+}
```

Comment:
I think `switch` statement can be used here as mentioned in this [post](https://www.oreilly.com/library/view/high-performance-javascript/9781449382308/ch04s02.html).

Reply:
I was thinking about, yes! Thank you!